Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,10 @@
 		57CDD9A02ADC31A8002EFED0 /* AddOdometerReadingView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 57CDD99F2ADC31A8002EFED0 /* AddOdometerReadingView.swift */; };
 		57CDD9A42ADC320F002EFED0 /* OdometerReading.swift in Sources */ = {isa = PBXBuildFile; fileRef = 57CDD9A32ADC320F002EFED0 /* OdometerReading.swift */; };
 		637505482AEFFBAC00AA5D0B /* FirebaseAnalytics+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 637505472AEFFBAC00AA5D0B /* FirebaseAnalytics+Extension.swift */; };
+		6D7232312B42F7D400E0EEC8 /* Date+extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6D7232302B42F7D400E0EEC8 /* Date+extension.swift */; };
```

Comment:
Have you considered using a build system like XcodeGen to help unclutter this kind of file?

Reply:
How is this cluttered?